### PR TITLE
Merge in-game alerts into single option

### PR DESCRIPTION
**Pull request by nwai90**

### CODE REVIEW

**Comment by alrusdi** [comment]
It is interesting. But what if i want overpayment warning, but don't want "mars is terraformed" notification?
More options - more flexibility isn't it?

**Comment by nwai90** [comment]
> It is interesting. But what if i want overpayment warning, but don't want "mars is terraformed" notification?
> More options - more flexibility isn't it?

True, this could also lead to having too many options though. Especially when there are more alerts added (e.g. Pass confirmation, other types of confirmations).

I'm fine with it both ways but might be too micro and perhaps overkill to be configuring each alert individually. Steam app also merges showing or hiding of alerts into a single config option which looks cleaner.

**Comment by alrusdi** [comment]
I'm not against it if everyone else like it. Let's wait a review from other contributors.

File: src/components/SelectHowToPay.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import Vue from "vue";
 import { HowToPay } from "../inputs/HowToPay";
 import { PaymentWidgetMixin } from "./PaymentWidgetMixin";
+import { PreferencesManager } from "./PreferencesManger";
```
Comment (bafolts) [inline src/components/SelectHowToPay.ts:5]: This file should be named `PreferencesManager` and not `PreferencesManger`.

Comment (nwai90) [reply src/components/SelectHowToPay.ts:5]: 👍 I renamed the file

**Pull request merged**